apps/journeys-admin/src/components/JourneyList/JourneyCard/JourneyCard.tsx

```diff
@@ -84,24 +112,94 @@ export function JourneyCard({
         >
           <CardActionArea
             disabled={isNavigating}
-            sx={{ borderRadius: 0, opacity: isNavigating ? 0.5 : 1 }}
+            sx={{
+              borderRadius: 0,
+              opacity: isNavigating ? 0.5 : 1,
+              flex: 1
+            }}
           >
-            <CardContent sx={{ px: 6, py: 4 }}>
-              <JourneyCardText journey={journey} variant={variant} />
+            {variant === JourneyCardVariant.new && (
+              <Box sx={{ position: 'absolute', top: 20, left: 20, zIndex: 1 }}>
+                <Chip
+                  label={t('New')}
+                  size="small"
+                  data-testid="new-journey-badge"
+                  sx={{
+                    backgroundColor: 'warning.main',
+                    color: 'warning.contrastText',
+                    borderRadius: 12.5,
+                    height: 25,
+                    px: 0.5,
+                    border: 1.5,
+                    borderColor: '#eb5b3b'
+                  }}
+                />
+              </Box>
+            )}
+            <Box
+              sx={{
+                position: 'relative',
+                width: 'auto',
+                paddingTop: {
+                  xs: `${(1 / 2) * 100}%`, // 1:2 aspect ratio - shorter social media image allows for more cards to be fitted into phone
```

Comment:
NIT:

```suggestion
                  xs: `50%`, 
```

thoughts on this?

Reply:
Yup I think that's allgoods! Making the change now.